libsolidity/parsing/Token.cpp

```diff
@@ -137,6 +145,25 @@ tuple<Token::Value, unsigned short, unsigned short> Token::fromIdentifierOrKeywo
 					return make_tuple(Token::IntM, m, 0);
 			}
 		}
+		else if (keyword == Token::UFixed || keyword == Token::Fixed)
+		{
+			auto positionN = find_if_not(positionX++, _literal.end(), ::isdigit);
```

Comment:
I'm not sure it actually matters...? I mean I can change it to that, but I don't think it will make a difference, unless you're aware of something I'm not.